cmd/manager/main.go

```diff
@@ -41,7 +41,7 @@ type studyCh struct {
 	addMetricsCh chan string
 }
 type server struct {
-	wIF         worker_interface.WorkerInterface
+	wIF         worker.WorkerInterface
```

Comment:
Can we change `WorkerInterface` to `Interface`, per
https://github.com/golang/go/wiki/CodeReviewComments#package-names

example:
https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go